civihr_employee_portal/src/View/CustomFieldsMappingFixer.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\View;
+
+/**
+ * To match unknown table names and field names when importing views
+ * with custom fields and custom groups from CiviCRM.
+ * Replaces unknown fields names and unknown tables like:
+ * - exported [name_of_the_field]_[ID1] with existing [name_of_the_field]_[ID2]
+ * - exported [name_of_the_table]_[ID3] with existing [name_of_the_table]_[ID4]
+ * ... where ID1, ID2, ID3 & ID4 are different numbers created by different
+ * mechanisms thus producing different IDs.
+ */
+class CustomFieldsMappingFixer {
+
+  /**
+   * To fix mapping on Views import of CiviCRM custom fields
+   *
+   * @param view $view
+   *     the view that is being imported / recreated
+   */
+  static public function fixMapping($view) {
+    // iterating displays in view
+    foreach ($view->display as $display) {
+      // iterating options on display: fields, filters, arguments
+      foreach ($display->display_options as &$displayOption) {
+        if (!is_array($displayOption)) {
+          continue;
+        }
+        self::processDisplayOptions($displayOption);
+      }
+    }
+  }
+
+  /**
+   * Process all display options for an exported view only when it is related
+   * to a field and the field has a number at the end of its name, which is
+   * the pattern for custom fields from CiviCRM
+   *
+   * @var array
+   *     Display option a exported view structure
+   */
+  static private function processDisplayOptions(&$displayOption) {
+    foreach ($displayOption as &$option) {
+      // only continue if this option is about a field
+      if (!isset($option['field'])) {
+        continue;
+      }
```

Comment:
Please add an empty line between logical blocks of code to improve readability. This applies to all the other methods on this class